tests/tests.js

```diff
@@ -65,3 +65,5 @@ Object.defineProperty(MyType.prototype, '0', {
 	}
 });
 assertDeepEquals(Array.of.call(MyType, 'abc'), { '0': 'abc', 'length': 1 });
+
+assertEquals(Array.of.name, 'of', 'Array#of has name "of"');
```

Comment:
this test will fail in IE, which doesn't have the name property - can you wrap it in an `if` that checks if function names are supported? My typical test is `(function foo() {}).name === 'foo'`.

(also, please add a trailing newline at the end of the file)